src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -61,6 +83,28 @@ private void bindDrivebaseControls() {
 		driveController.rightStick().onTrue(new InstantCommand(s.drivebaseSubsystem::toggleXWheels));
 	}
 
+	// intake controls
+	private void bindIntakeControls() {
+		// later on set default command to check sensing notes
+		CommandScheduler.getInstance()
+				.setDefaultCommand(s.intakeSubsystem, new IntakeStopCommand(s.intakeSubsystem));
```

Comment:
Because `IntakeOutCommand` ends immediately, the intake motors will only run briefly before `IntakeStopCommand` makes them all stop. I think not having a default command is ok for now especially since this one is just a placeholder.